src/tests/unit/connection.test.js

```diff
@@ -24,6 +22,7 @@ describe('Test oracledb connection module', () => {
       .withArgs('dataSources.oracledb')
       .returns({});
   });
+  afterEach(() => sinon.restore());
 
   const createOracleDbStub = (createPoolStub) => {
     connection = proxyquire('api/v1/db/oracledb/connection', {
```

Comment:
I think this file is not needed since Notes API only connect to AWS S3. Sorry for not catching it previously.